.github/ISSUE_TEMPLATE/Django Feature.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Django Feature Request
+about: Suggest an idea for this project
+title: ""
+labels: 'Django feature'
+assignees: ''
+
+---
```

Comment:
I don't think we need this kind of a table

Reply:
I have removed the table